reference-implementation/lib/abstract-ops/readable-streams.js

```diff
@@ -1263,18 +1289,17 @@ function ReadableByteStreamControllerRespondInReadableState(controller, bytesWri
 
 function ReadableByteStreamControllerRespondInternal(controller, bytesWritten) {
   const firstDescriptor = controller._pendingPullIntos[0];
+  assert(CanTransferArrayBuffer(firstDescriptor.buffer) === true);
```

Comment:
`IsDetachedBuffer` is sufficient here. Both `ReadableByteStreamControllerRespond` and `ReadableByteStreamControllerRespondWithNewView` will have called `TransferArrayBuffer` already.

Reply:
I'd be happy leaving CanTransferArrayBuffer in the asserts, for spec clarity.